addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -148,7 +148,7 @@ export const CLIPBOARD_WHITELISTS = {
         /^btn/,
         /^fa/,
     ],
-    attributes: ['class', 'href', 'src'],
+    attributes: ['class', 'href', 'src', 'target'],
```

Comment:
Note that I'm not against having separate parts for "previously" and "this commit fixes that". In the case of this commit for example, I like it because we have interesting things to say in the "this commit fixes that" part. This was not really the case in the previous commit outside of stating the obvious "we fix the selector". Here otherwise, it's not that easy from the diff itself to understand where the change is made and why it does have the desired effect, so the commit message has more value. 👍 

I still have two remarks though:
- The third sentence is missing a dot at the end.
- Loss of information is not a big deal per se so I don't think I'd mention it like this but if you really like it you can keep it. However, what does matter and is not in the commit message, is _why_ losing the link target is a big deal. What does it change functionally ? The functional issue that came from losing the target attribute is that links that were supposed to open in a new tab were transformed into links that open in the current tab when pasted into the editor. This is important information for two reasons IMO. First, because it is the behavior that we want to fix. It's not like we want to keep all attributes and lose no information on paste, no no, we lose plenty of information on paste and it's on purpose because we _do_ want to sanitize what is pasted. The problem here is that pasting a link used to functionally change how the link behaved and _that_ is why we wanted to fix this. The second reason is because for people who are not familiar with web development, they might not know what effect losing the target attribute might have. Imagine someone who might want to revert your fix in a few years because they have a weird bug case where they think reverting your commit might fix it. From the commit message, they cannot know what behavior you were trying to fix besides keeping the target attribute. They might revert your commit and see that the link works even without a target attribute so why keep it if it triggers a bug somewhere else ? They might revert your commit and reintroduce the original bug without realizing it. They wouldn't know it because the commit message does not explain what the actual issue was.

Reply:
Let me know what you think about my changes :)

Reply:
Better, thanks ! :) I'm r+ing this PR. 🚀